### ISSUE TITLE
Accessibility issues in Login Page

### ISSUE THREAD

**Issue by nirajCITZ** [bcgov/api-services-portal#795]
1)
Description:  Ensures the contrast between foreground and background colors meets WCAG 2 AA minimum contrast ratio thresholds
Helpurl:      https://dequeuniversity.com/rules/axe/4.7/color-contrast?application=axeAPI
[
    {
        "any": [
            {
                "id": "color-contrast",
                "data": {
                    "fgColor": "#4d6d96",
                    "bgColor": "#657fa5",
                    "contrastRatio": 1.3,
                    "fontSize": "12.0pt (16px)",
                    "fontWeight": "normal",
                    "messageKey": null,
                    "expectedContrastRatio": "4.5:1"
                },
                "relatedNodes": [
                    {
                        "html": "<a data-testid=\"login-api-provider-btn\" href=\"http://oauth2proxy.localtest.me:4180/login?identity=provider&amp;f=%2F\" id=\"menu-list-15-menuitem-12\" role=\"menuitem\" tabindex=\"0\" class=\"chakra-menu__menuitem css-xe49rg\" data-index=\"0\">",
                        "target": [
                            "#menu-list-15-menuitem-12"
                        ]
                    },
                    {
                        "html": "<div tabindex=\"-1\" role=\"menu\" id=\"menu-list-15\" aria-orientation=\"vertical\" class=\"chakra-menu__menu-list css-1f8elly\" style=\"transform-origin: var(--popper-transform-origin); opacity: 0.236571; visibility: visible; transform: scale(0.847314) translateZ(0px);\">",
                        "target": [
                            "#menu-list-15"
                        ]
                    },
                    {
                        "html": "<nav role=\"banner\" class=\"css-f4yi47\"><ul class=\"chakra-container css-u78s0k\"><li class=\"css-c92nq2\"><a class=\"chakra-link css-8oz1d\" aria-current=\"false\" data-testid=\"navbar-link-API Directory\"><span class=\"css-1o39rat\">API Directory</span></a></li></ul></nav>",
                        "target": [
                            "nav"
                        ]
                    }
                ],
                "impact": "serious",
                "message": "Element has insufficient color contrast of 1.3 (foreground color: #4d6d96, background color: #657fa5, font size: 12.0pt (16px), font weight: normal). Expected contrast ratio of 4.5:1"
            }
        ],
        "all": [],
        "none": [],
        "impact": "serious",
        "html": "<span style=\"pointer-events: none; flex: 1 1 0%;\">API Provider</span>",
        "target": [
            "#menu-list-15-menuitem-12 > span:nth-child(2)"
        ],
        "failureSummary": "Fix any of the following:\n  Element has insufficient color contrast of 1.3 (foreground color: #4d6d96, background color: #657fa5, font size: 12.0pt (16px), font weight: normal). Expected contrast ratio of 4.5:1"
    }
]
![image.png](https://images.zenhubusercontent.com/61a121c4a7a3264dfdc530ef/03205c25-630c-4ae6-a41f-f17273ccbc79)

**Comment by veep-22** [comment]
linked to APS-2185 on JIRA https://dpdd.atlassian.net/browse/APS-2185
